aws/resource_aws_pinpoint_app.go

```diff
@@ -128,6 +128,12 @@ func resourceAwsPinpointApp() *schema.Resource {
 					},
 				},
 			},
+			"arn": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+			},
+			"tags": tagsSchema(),
```

Comment:
This new argument is missing documentation in `website/docs/r/pinpoint_app.html.markdown`, e.g. (under `## Arguments`)

```
* `tags` - (Optional) Key-value mapping of resource tags
```

Reply:
Added 👍